torch_frame/datasets/data_frame_benchmark.py

```diff
@@ -539,7 +539,7 @@ class DataFrameBenchmark(torch_frame.data.Dataset):
           - Yandex(name='year')
           - 0.0%
     """
-    dataset_categorization_dict = {
+    dataset_categorization_dict: dict[str, dict[str, list[tuple]]] = {
```

Comment:
Just curious what are the benefits of adding this type hint. Would the type hint test break if someone adds a dataset that doesn't follow the type hint here?

Reply:
Yes, it'll break the type check test, but I don't think there'll be a case when the type is different than this.